app/models/ability.rb

```diff
@@ -127,6 +127,7 @@ def signed_in_with_organizer_role(user)
     can :manage, Vday, conference_id: conf_ids_for_organizer
     can :manage, CallForPaper, conference_id: conf_ids_for_organizer
     can :manage, Event, conference_id: conf_ids_for_organizer
+    can :schedule, Event, conference_id: conf_ids_for_organizer
```

Comment:
Yes indeed, and in fact there does not even exist a function called 'schedule' in the controller. 
The reason I thought we could do it like this is merely for clarity.
You can schedule an event if you can update its start time. Perhaps that is not 100% clear to everyone.
If you think it does not really add any value, I ll remove it.